.travis.yml

```diff
@@ -38,5 +39,3 @@ matrix:
       env: TASK=ataru
     - rvm: 2.2
       env: TASK=mutant
-    - rvm: 2.2
-      env: TASK=rubocop
```

Comment:
not interested. Until Rubocop allows for new cops to be opt-in instead of opt-out I am not bothering with it.